cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ClusterCa.java

```diff
@@ -241,4 +241,19 @@ public Secret cruiseControlSecret() {
             isMaintenanceTimeWindowsSatisfied);
     }
 
+    @Override
+    protected String caCertThumbprintAnnotation() {
+        return "cluster-ca-thumbprint";
```

Comment:
Should this be defined as constanct somewhere in `Annotations`?

Reply:
Yeah it makes sense even if I preferred to add the annotations this way in the `Ca` where other CA management related annotations live:

```java
public static final String ANNO_STRIMZI_IO_CLUSTER_CA_THUMBPRINT = Annotations.STRIMZI_DOMAIN + "cluster-ca-thumbprint";
    public static final String ANNO_STRIMZI_IO_CLIENTS_CA_THUMBPRINT = Annotations.STRIMZI_DOMAIN + "clients-ca-thumbprint";
```

Reply:
If the other annos are there already then yea, it might make more sense to add them there.